scripts/android-build-freerdp.sh

```diff
@@ -37,6 +38,11 @@ do
 			WITH_OPENH264=1
 			shift
 			;;
+		--openh264-ndk)
+			shift
+			ANDROID_NDK_OPENH264=$1
```

Comment:
Setting this to `ANDROID_NDK` by default would be a nice to have

Reply:
Have a look further down the same file where this is already done:

```patch
@@ -107,11 +112,19 @@ do
     fi
     if [ $WITH_OPENH264 -ne 0 ];
     then
+        if [ -z "$ANDROID_NDK_OPENH264" ]
+        then
+            echo
+            echo "Warning: Missing openh264-ndk, using $ANDROID_NDK" >&2
+            echo
+            ANDROID_NDK_OPENH264=$ANDROID_NDK
+        fi
         if [ $BUILD_DEPS -ne 0 ];
         then
             common_run bash $SCRIPT_PATH/android-build-openh264.sh \
                 --src $BUILD_SRC/openh264 --dst $BUILD_DST \

```

Reply:
oh, sorry, missed that :)